Documentation/Upgrade/rook-upgrade.md

```diff
@@ -243,4 +243,9 @@ This cluster is finished:
 
 At this point, the Rook operator should be running version `rook/ceph:v1.12.0`.
 
+!!! note
+	In case of using the [Rook toolbox](https://rook.io/docs/rook/latest/Troubleshooting/ceph-toolbox/), users must
```

Comment:
Since the toolbox is based on the Ceph image, this will fit better in ceph-upgrade.md. In [that doc](https://rook.io/docs/rook/latest/Upgrade/ceph-upgrade/#example-upgrade-to-ceph-quincy) we have three sections, I'd suggest making it four sections:
1. Update the Ceph daemons
2. Update the toolbox image
3. Wait for the pod updates
4. Verify cluster health

Reply:
> Since the toolbox is based on the Ceph image, this will fit better in ceph-upgrade.md. In [that doc](https://rook.io/docs/rook/latest/Upgrade/ceph-upgrade/#example-upgrade-to-ceph-quincy) we have three sections, I'd suggest making it four sections:
> 
> 1. Update the Ceph daemons
> 2. Update the toolbox image
> 3. Wait for the pod updates
> 4. Verify cluster health

Thanks @travisn for pointing me out to these docs. Indeed it seems as good place to mention the toolbox but I still think adding a note about the toolbox in the rook-upgrade could help users upgrading rook to keep it mind. Otherwise they will end up with a mismatch version between the ceph Image used by the operator and the one in use the toolbox. This could cause issues as the linked in this PR. Since we are not making any special reference (it's just a note) it could save users time and avoid potential issues.

WDYT?

Reply:
@travisn I see, was thinking in the case where somebody upgrades Rook to a new operator version (which comes with a new ceph-version) but then forget to upgrade the toolbox to this new version.